DEPLOYMENT_PROCEDURE.md

```diff
@@ -4,23 +4,26 @@ This document is only intended for Pathoplexus sysadmins.
 
 ## To develop new versions of Pathoplexus
 
-Work in a development branch. You can choose what version of Loculus to target by adjusting the `loculusVersion` in  `pathoplexus_app/values.yaml`. Use a Loculus commit SHA.
+Work in a development branch. You can choose what version of Loculus to target by adjusting the `loculusVersion` in  `pathoplexus_app/values.yaml` (use a Loculus commit SHA). Adding the `preview` label will create a preview.
 
-## Merging to main
+## To simply bump the version of Loculus used in Pathoplexus `main`
+We have a GitHub workflow for this. Click Actions -> Create PR to bump main pathoplexus.. -> Run workflow -> Run workflow. This will create a PR, which someone else will need to approve. The PR will contain a changelist for Loculus, which you should carefully review for breaking changes.
```

Comment:
```suggestion
We have a GitHub workflow for this. Go to the "Actions" tab -> Open the "Create PR to bump main pathoplexus.." action -> Open the "Run workflow" dropdown -> Click on the "Run workflow" button. This will create a PR, which someone else will need to approve. The PR will contain a changelist for Loculus, which you should carefully review for breaking changes.
```